sdk-core/src/main/java/com/sportradar/unifiedodds/sdk/OddsFeed.java

```diff
@@ -144,10 +158,13 @@ public OddsFeed(SDKGlobalEventsListener listener, OddsFeedConfiguration config)
         this.injector = createSdkInjector(listener, null);
         checkLocales();
         this.oddsFeedExtListener = null;
+        this.concurrentListenerConfig = config.getConcurrentListenerConfig();
```

Comment:
Each constructor needs to init the global listener (which may be the Concurrent Global Listener depending on config)